mantle/platform/qemu.go

```diff
@@ -2063,3 +2064,12 @@ func (builder *QemuBuilder) Close() {
 		os.RemoveAll(builder.tempdir)
 	}
 }
+
+// supports IBM Cex based LUKS encryption if it is s390x host (zKVM/LPAR)
+func (builder *QemuBuilder) AddCexDevice() error {
+	if builder.architecture != "s390x" {
```

Comment:
I don't think we need to guard on this. It should only be called on s390x.

Reply:
Will correct it